Tests/testconfig.ini

```diff
@@ -40,3 +40,7 @@ baudrate=9600
 [UdpAddress]
 ipAddress=239.255.43.21
 port=44550
+
+[rabbitMQ]
+routingKey=10vby5v5j3orid8
```

Comment:
Completely arbitrary, I just chose something that would hopefully never be randomly created by anyone else using the same rabbitmq server for something else